src/csharp/ext/std++compat.cc

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <cstdarg>
+#include <cstdio>
+#include <vector>
+
+#if defined(__GNUC__)
+
+namespace std {
+
+// CentOS 7 (GLIBC_2.17 / GLIBCXX_3.4.19) doesn't have a following symbol
```

Comment:
what's the minimum GLIBCXX required?

Reply:
We don't have version requirement for GLIBCXX in general. This happens to have this because .NET artifacts are built on Debian 8. If we're using Cent OS 6 or 7 to build .NET artifacts, then it has a lower version requirement for GLIBCXX, which is the same idea with the manylinux platform for Python.

Reply:
Btw, we could build the .NET artifacts on an older distro if needed.

Reply:
This doesn't prevent users from building .NET artifacts from source as far as they have required build tools available.